### PR TITLE
Release 1.0.5

### PR DESCRIPTION
**Pull request by ddl-alexpanin**
Release 1.0.5.

### CODE REVIEW

File: domino/_version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '1.0.4'
+__version__ = '1.0.5'
```
Comment (dknupp) [inline domino/_version.py:1]: A discussion with Med in a Slack thread came up re: changing the version scheme of this package to match the corresponding Domino version. A user would know immediately, for example, that to access a API features associated Domino 4.5.0, they would need to pip install `python-domino==4.5.0`.

@mpkouznetsov @dmitriyvolk -- any thoughts?

**Comment by dknupp** [comment]
> Also will be publishing releases to PyPI.

One general thing to note @ddl-alexpanin about PyPI. If you release a package versioned as 1.2.3, that number can never be re-used. In other words, if you notice an error after the fact, correct it, and then try to re-upload version 1.2.3, you will be prevented from doing so.

Any change, even if it's just a typo fix in the README, will require a new version (e.g., 1.2.4).

This is a security feature so that no package can ever be surreptitiously replaced with code that mimics it, but also perhaps does something harmful.

**Comment by ddl-alexpanin** [comment]
> One general thing to note @ddl-alexpanin about PyPI. If you release a package versioned as 1.2.3, that number can never be re-used. 

Yes, this is why https://test.pypi.org/project/python-domino/ is already 1.0.13 :)

**Comment by dknupp** [comment]
> Yes, this is why https://test.pypi.org/project/python-domino/ is already 1.0.13 :)

👍

File: domino/_version.py (hunk 1 above)
Comment (mpkouznetsov) [reply domino/_version.py:1]: No strong opinion. python-domino supports multiple versions of domino so aligning the versioning schemes will sort of hint that we support _only_ the latest version. Still, it probably makes more sense to align.

Comment (ddl-alexpanin) [reply domino/_version.py:1]: Discussed with Pablo and we decided not to change versioning scheme at this moment.

**Review by mpkouznetsov — APPROVED** [review]
LGTM

**Pull request merged**